engineapi/instance_manager.go

```diff
@@ -216,6 +220,9 @@ func (c *InstanceManagerClient) ReplicaProcessCreate(replicaName, engineImage, d
 	if revCounterDisabled {
 		args = append(args, "--disableRevCounter")
 	}
+	if enableUnmapMarkDiskChainRemoved {
+		args = append(args, "--unmap-mark-disk-chain-removed")
+	}
```

Comment:
Need to check the cli version, or the volume with an old engine cannnot be started because of the `--unmap-mark-disk-chain-removed`. missing error of.